### PR TITLE
add authentication and routes 

### PR DESCRIPTION
**Pull request by archit7448**
## What to check : 
- [x] Sign In jsx 
- [x] All jsx file

### CODE REVIEW

**Review by Muzammil-khan710 — APPROVED** [review]
Great UI
Clean code !
Good to merge

File: src/pages/Homepage/HomePage.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Header } from "../../Components/header/header";
 import mainImage from "../../assets/homepage.png";
-import { Genres } from "../../Components/genres/categories";
+import { Genres } from "../../Components/genres/cateories";
```
Comment (iRohitGaur) [inline src/pages/Homepage/HomePage.jsx:3]: suggestion: spelling of categories

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by zoef-yt — APPROVED** [review]
The code looks clean. The UI looks great. Good to merge nice work.

File: src/pages/Homepage/HomePage.jsx (hunk 1 above)
Comment (archit7448) [reply src/pages/Homepage/HomePage.jsx:3]: sure  I will update in next pr

**Pull request merged**